FlowerOrders/FlowerOrders/Screens/OrdersList/Cell/OrderTableViewCell.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  OrderTableViewCell.swift
+//  FlowerOrders
+//
+//  Created by Paul Tiriteu on 05/08/2020.
+//  Copyright © 2020 Paul Tiriteu. All rights reserved.
+//
+
+import UIKit
+import Alamofire
```

Comment:
Do we need Alamofire here ? we also don't use it in the Networking class.

Reply:
We don't need Alamofire at all. Rookie mistake